arch/arm/core/aarch32/Kconfig

```diff
@@ -215,6 +215,34 @@ config ARM_NONSECURE_FIRMWARE
 	  resources of the Cortex-M MCU, and, therefore, it shall avoid
 	  accessing them.
 
+config ARM_STORE_EXC_RETURN
+	bool
+	default y if USERSPACE || FPU_SHARING || ARCH_HAS_TRUSTED_EXECUTION
+	help
+	  Store the EXC_RETURN value when switching threads.
+	  This is needed when switching between threads that differ in either
+	  FPU usage or security domain.
+
+config ARM_DEFAULT_THREAD_EXC_RETURN
```

Comment:
Why does it need to be a Kconfig?
Could we just define the values in arch.h for Cortex-M?

Reply:
if we decide to keep it here: could we add a depends on ARM_STORE_EXC_RETURN?